src/main/java/org/folio/roles/integration/kafka/CapabilityEventProcessor.java

```diff
@@ -99,7 +100,7 @@ private CapabilityResultHolder processUiModuleResources(CapabilityEvent event, L
   private Optional<CapabilitySetDescriptor> createCapabilitySetDescriptor(
     CapabilityEvent event, FolioResource resource, ModuleType moduleType) {
     var folioPermission = resource.getPermission().getPermissionName();
-    var permissionMappingOverrides = emptyIfNull(event.getPermissionMappingOverrides());
+    var permissionMappingOverrides = permissionOverrider.getPermissionMappings();
```

Comment:
cant be null?

Reply:
No, PermissionOverrider either returns result of `return permissions.entrySet().stream().collect(toMap(Map.Entry::getKey, PermissionOverrider::mapToPermissionData));` or throws exception.